### PR TITLE
summer techtalk report

### PR DESCRIPTION
**Pull request by federica-buzzi**

### CODE REVIEW

File: recap-of-techtalks-summer-edition-2021.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+category: "techtalks"
+tags:
+  - post
+  - guest-post
+layout: article
+metaIMG: "womendotcodebe-techtalks-recap-june2021.jpg"
+featured: true
+---
+
+Buckle up for the recap of another eclectic Techtalk.
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: Maybe turn int into "eclectic edition of our TechTalks"

File: recap-of-techtalks-summer-edition-2021.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+Computing technologies are part of the daily life of 5 billions of users with 34 billions of connected devices, and it’s estimated that these numbers will double by 2025! 😱
+
+Also the environmental costs of IT’s ever-expanding presence are skyrocketing. Here are some figures:
+
+- The IT industry produces 4% of the greenhouse gas emissions in the world (more than the aviation sector which is responsible only for 2% of emissions)
+- It takes 10% of the world-wide electricity consumption
+- It is responsible for abiotic resources depletion
+- It perpetrates inhumane working conditions. In the Democratic Republic of Congo 6 million of workers died in coltan mines, a material used to produce electronics.
+
+Let’s break the IT industry into its component to understand why it has such a big ecological impact.There are three main actors in IT:
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: its component <> smaller components?

File: recap-of-techtalks-summer-edition-2021.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+1. The end user device
+2. The network infrastructure
+3. The data centres
+
+We can identify two sources of impact:
+
+1. The manufacture of equipments
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: Lose the s at the end of equipment

File: recap-of-techtalks-summer-edition-2021.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+2. The manufacture of electricity
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md:47]: Not sure if manufacture is the right word here.

The process of generating electricity?

File: recap-of-techtalks-summer-edition-2021.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+Even if data centres conjure the image of the most energy-consuming part of IT industry, it is the end user devices that are responsible for most of the environmental impact - and especially their manufacture!
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: centres <> centers

Comment (claudia-rndrs) [reply recap-of-techtalks-summer-edition-2021.md]: Depending if you use british English or American English throughout the recap :)

File: recap-of-techtalks-summer-edition-2021.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+
+1. TV (14%)
+2. Desktop (7,5%)
+3. Laptop (7%)
+
+Maybe you too were expecting to see the smartphone showing up as number one? The reason why it doesn’t is that this ranking takes into account impact due to both manufacturing and use. With 4,5% the smartphone only comes sixth in the list but here’s the caveat: while it has a negligible impact in terms of usage (it doesn’t need to be plugged to energy to function) the smartphone has the highest impact for manufacturing only.
+
+### What can we do as citizen?
+
+As citizens we can strive towards a digital sobriety approach: try to buy less equipment which lasts longer and is really useful.
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: This is a bit of a confusing sentence. Now it sounds like you should buy less equipment that lasts longer and is really useful. 

But I think you're trying to say that we should buy less equipment, but when we do it should focus on something that lasts longer and is really useful, no?

File: recap-of-techtalks-summer-edition-2021.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+
+The outcome of a research on the use of disability settings by the dutch company [Q42](https://werkenbij.q42.nl/) shows it well. The most used disability feature is reciting tex (33%), which makes sense in relation to the visually impaired or the elderly. But the second most used feature is an interesting reveal: dark mode (27%) used to prevent tired eyes and increase contrast, which illustrates the aforementioned broad spectrum of disabilities and how accessibility can also benefit people without disabilities.
+
+### WCAG (Web Content Accessibility Guidelines)
+
+We could choose to eco-design a digital service, which means trying to decrease its environmental impact by improving its conception and realisation. Which is basically a twofold operation: on one hand trying to increase the lifetime of end user devices, on the other trying to decrease the physical resources needed to run our software.
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: conception <> concept

File: recap-of-techtalks-summer-edition-2021.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+#### Some tips & tools
+
+[To calculate accessibility score of your website](https://wave.webaim.org/)
+[To share files](https://filevert.fr/)
+[For secured visio-conferences](https://www.infomaniak.com/fr/kmeet)
+[To avoid WordPress to create a simple website](https://gohugo.io/)
+To repair your devices : [Repaircafe](https://repaircafe.org/fr/) & [Fixware](https://www.fixware.be/)
+[To buy reconditioned devices](https://www.asmartworld.be/)
+
+#### Asma Oualmakran - Application security: Finding your way through buzzwords
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: wrong header, should be a h2


Best to review all headers for Asma's talk

File: recap-of-techtalks-summer-edition-2021.md [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+#### The developer's security toolkit
+
+- **Security code scanning tools** Checkmarx, Veracode, Fortify, Coverity, Sigrid
+- **Opensource health** Snyk, Blackduck, Whitesource, Sigrid
+- **Maintainability measurement** SonarQube, Sigrid
+- **Process maturity guidelines and testing** OWASP SAMM , BSIMM, CIP
+- **Training tools** Secure code warrior, Secure Flag, Codebashing
+- **Security references** NVD, MITRE, ASVS, OWASP Top 10
+
+## Liza Morrison - Be Qwick, don’t ship JavaScript
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: Qwick should be Qwik (you repeated the typo throughout the recap)

**Review by claudia-rndrs — CHANGES_REQUESTED** [review]
Good work! 

I did add some comments to what might need to be changed.
Apart from that I think the only thing that might be missing is an endword :)

File: recap-of-techtalks-summer-edition-2021.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+
+The harsh truth is that 40% of visitors are going to leave your page if that doesn’t display correctly within the first 3 seconds.
+
+Today speed is everything, especially on the web. Not only for user experience and for those who are constantly in a hurry: having a top performance score is also going to improve the SEO (Search Engine Optimisation) of your page.
+
+Time To Interactive (TTI), that is how long it takes your page to become fully interactive, is also an important metric for performance.
+
+### Ship less JavaScript
+
+When a website scores poorly on performance the blame falls on the the developer, but what is it was the framework’s fault instead?
```
Comment (claudia-rndrs) [inline recap-of-techtalks-summer-edition-2021.md]: A typo: "but what is" should be what if

**Review by claudia-rndrs — APPROVED** [review]

**Comment by claudia-rndrs** [comment]
@federica-buzzi A bit of extra feedback: 
- In the future if you use PDF's don't add them with the image script tags, this will result in several issues with the image optimisation script that is in place. You can add regular html tags in Markdown instead. But in this case it's probably better not to use PDF.

- In the future please add some comments for the parts I still need to fill in 😅 (for example the speaker images) otherwise I might forget.

I am currently doing a final check locally before your recap goes life, so don't push any more changes! :)

**Comment by federica-buzzi** [comment]
gotcha!

**Pull request merged**